docker/dev/clone_db.sh

```
@@ -0,0 +1,42 @@
+# Remove the dump file to clean up
+echo "Removing the temporary dump file from the container..."
+docker exec journeyplanner-db-1 bash -c 'rm /var/lib/mysql/journeyplanner.sql'
+
+# Run database migrations to ensure schema is up to date
+echo "Running database migrations to apply any pending changes..."
+cd ../../backend
+infisical run --env=dev -- php artisan migrate
+
+# Stop the database container
+echo "Stopping the database container to free up resources..."
+cd ../docker/dev
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)